Glyssen/Dialogs/VoiceActorAssignmentDlg.cs

```diff
@@ -39,9 +39,9 @@ public VoiceActorAssignmentDlg(Project project)
 
 				foreach (KeyValuePair<int, CharacterGroup> pair in charGroupTemplate.CharacterGroups)
 					characterGroups.Add(pair.Value);
-			}
 
-			m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
+				m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
```

Comment:
Won't this make it so that the values won't be populated when returning to the grid ?